daemon/daemon_windows.go

```diff
@@ -24,6 +24,14 @@ func getBlkioWeightDevices(config *runconfig.HostConfig) ([]*blkiodev.WeightDevi
 	return nil, nil
 }
 
+func getBlkioReadBpsDevices(bpsDevices []string) ([]string, error) {
```

Comment:
I don't think we needs these definition in `_windows.go`. The caller of these functions is in a unix-only path. They are never called on windows, are they?